Packs/CortexResponseAndRemediation/Playbooks/playbook-WmiPrvSe.exe_Rare_Child_Command_Line.yml

```diff
@@ -0,0 +1,1365 @@
+
+        **Matched Verdicts:**
+        * Matches for high-risk command line analysis profiles: mixed_case_powershell, double_encoding, amsi_techniques, malicious_commands, or powershell_suspicious_network_patterns.
+
+        * Malicious IP address detected.
+        Score >= 25, indicating high confidence probability for malicious behavior.
+
+        * Score >= 10 with a prevention rule detected in the same alert, correlating to malicious activity.
+
+        **Action Required:**
+
+        * Isolate the remote host: ${Endpoint.Hostname}
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      Isolate:
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 451,
+          "y": 2670
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - Isolate
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 7df12c62-a960-428c-8e0f-dccf404b63e0
+    type: regular
+    task:
+      id: 7df12c62-a960-428c-8e0f-dccf404b63e0
+      version: -1
+      name: Isolate endpoint
+      description: Isolates the specified endpoint.
+      script: '|||core-isolate-endpoint'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      endpoint_id:
+        simple: ${Endpoint.ID}
+      incident_id:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 451,
+          "y": 2865
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: aa59de8b-dca5-485b-90db-49fc8bad71dc
+    type: condition
+    task:
+      id: aa59de8b-dca5-485b-90db-49fc8bad71dc
+      version: -1
+      name: Should proceed to isolate the remote endpoint?
+      description: If the condition "Check for high-confidence evidence or malicious IP address" was matched and the remote endpoint ID is available, an endpoint isolation is suggested.
```

Comment:
The description does not match the task conditions. Please update it to align with the condition terms.
For example, "malicious IP address" does not appear in the terms.
or
"remote endpoint ID"